gqlgen.yml

```diff
@@ -57,3 +58,6 @@ models:
   UUID:
     model:
       - github.com/cmsgov/mint-app/pkg/models.UUID
+  # UserAccount:
+  #   model:
+  #     - github.com/cmsgov/mint-app/pkg/authentication.UserAccount
```

Comment:
Per our review, might be good to just call out the single UserAccount bind instead of the entire `authentication` library.

On that note - could we move `UserAccount` into `pkg/models` and reference it from `authentication`? That way we'd only need the single autobind statement